lib/src/diktat/java/com/diffplug/spotless/glue/diktat/DiktatFormatterFunc.java

```diff
@@ -33,15 +33,13 @@
 public class DiktatFormatterFunc implements FormatterFunc.NeedsFile {
 
 	private final List<RuleSet> rulesets;
-	private final Map<String, String> userData;
```

Comment:
So `userData` is nuked in `1.2.0`, and anybody who had been using that should stop? If that's the case, that's okay, but it should be clearly noted in the changelogs, and `userData` should be marked deprecated and should throw an error in the Gradle and Maven DSL.

Reply:
yes, changing of `usedData` was prohibited, but diktat uses it only for internal logic (it was not supposed to be set by client).
will add note to the changelogs and add errors to plugins

Reply:
@nedtwigg, actually, I don't see calls of method from `DiktatStep` with `userData` parameter.
I only see calls:
1. `com.diffplug.spotless.kotlin.DiktatStep#create(java.lang.String, com.diffplug.spotless.Provisioner, com.diffplug.spotless.FileSignature)`:
    * `com.diffplug.gradle.spotless.KotlinExtension.DiktatFormatExtension#createStep` 
    * `com.diffplug.spotless.maven.kotlin.Diktat#newFormatterStep`
2. `com.diffplug.spotless.kotlin.DiktatStep#createForScript`: 
    * `com.diffplug.gradle.spotless.KotlinGradleExtension.DiktatFormatExtension#createStep`


So, it's safe to remove `userData`